src/containers/access/stories/SaveNewFormTemplate.stories.js

```diff
@@ -0,0 +1,24 @@
+import SaveNewFormTemplate from './components/SaveNewFormTemplate';
+
+import ModuleProvider from '../../../core/provider/ModuleProvider';
+import { schemas, uiSchemas } from '../../../forms/common/schemas';
+
+export default {
+  title: 'Access Request/Save New Form Template',
+  component: SaveNewFormTemplate
+};
+
+/* eslint-disable react/jsx-props-no-spreading */
+const Template = (args) => (
+  <ModuleProvider>
+    <SaveNewFormTemplate {...args} />
+  </ModuleProvider>
+);
+
+/* eslint-disable max-len */
```

Comment:
This looks like an unnecessary disable.

Reply:
the `react/jsx-props-no-spreading` seems necessary. but yeah, the `/* eslint-disable max-len */` should be replaced with `/* eslint-enable */`